sdk/reservations/Azure.ResourceManager.Reservations/src/autorest.md

```diff
@@ -7,7 +7,7 @@ azure-arm: true
 csharp: true
 library-name: Reservations
 namespace: Azure.ResourceManager.Reservations
-require: https://github.com/Azure/azure-rest-api-specs/blob/42f123a0ca6cd5f8f01f3463ecb47999fdbf3a18/specification/reservations/resource-manager/readme.md
+require: https://github.com/Azure/azure-rest-api-specs/blob/04efa7dde730c9eaa35bdab1681117d639522edb/specification/reservations/resource-manager/readme.md
 tag: package-2022-03
```

Comment:
Update the spec without version change?

Reply:
Yes, because our team is planning to add the two sets of APIs to the existing latest stable version, and this breaking change has been approved by ARM.